data/announcements/scheduled.yaml

```diff
@@ -174,3 +174,23 @@ announcements:
     k8s.gcr.io image registry is gradually being redirected to registry.k8s.io (since Monday March 20th).<br>
     All images available in k8s.gcr.io are available at registry.k8s.io.</br>
     Please read our [announcement](/blog/2023/03/10/image-registry-redirect/) for more details.
+
+- name: Freezing the legacy package repositories - Before
+  startTime: 2023-09-01T00:00:00
+  endTime: 2023-09-13T19:59:59
+  style: "background: #c70202"
+  title: Changes to Linux package downloads for Kubernetes
```

Comment:
Opinion
I feel like this is a little too concise to catch the right attention. Here's two other options:
"Linux packages for Kubernetes installs are changing"
"Changes to the location of Linux packages for Kubernetes"

Reply:
+1 to "Changes to the location of Linux packages for Kubernetes"

Reply:
Initially it was "The legacy package repositories will be frozen starting with September 13, 2023" which is kind of too long and maybe we can do something like "The legacy Linux package repositories are getting frozen" as well. I don't have strong opinions.

Reply:
Changed to "Changes to the location of Linux packages for Kubernetes"